src/ccnl-utils/src/ccn-lite-rpc.c

```diff
@@ -175,6 +175,10 @@ parsePrefixTerm(int lev, char **cpp)
                     free(term);
                 }
 
+                if (t2) {
```

Comment:
is the check to `t2` really necessary? It seems that this is checked a couple of lines above?

Reply:
I prefer to check if something is actually non-null before I try to free it. If you want we can remove the check, but given the "complexity" of the function I would prefer to leave it there.